File: Estrutura_Repetição/Clojure/Ex1074.clj
(dotimes [n (read)]
  (let [X (read)]
    (cond
      (== X 0) (println "NULL")
      (odd? X) (print "ODD ")
      (even? X) (print "EVEN "))
    (cond
      (> X 0) (println "POSITIVE")
      (< X 0) (println "NEGATIVE"))))

File: Estrutura_Repetição/Python/Ex1074.py
valores = int(input())

for i in range(valores):
    v = int(input())

    if v != 0:
        if v % 2 == 0:
            if v > 0:
                print("EVEN POSITIVE")
            else:
                print("EVEN NEGATIVE")
        else:
            if v > 0:
                print("ODD POSITIVE")
            else:
                print("ODD NEGATIVE")
    else:
        print("NULL")